src/migration/1670956536231-new_records.ts

```diff
@@ -0,0 +1,117 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+import config from '../config';
+
+export class newRecords1670956536231 implements MigrationInterface {
+  name = 'newRecords1670956536231';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "iasql_database" DROP COLUMN "operation_count"`);
+    await queryRunner.query(`ALTER TABLE "iasql_database" DROP COLUMN "rpc_count"`);
+    await queryRunner.query(
+      `ALTER TABLE "iasql_database" ADD "records_applied" integer NOT NULL DEFAULT '0'`,
+    );
+    await queryRunner.query(`ALTER TABLE "iasql_database" ADD "records_synced" integer NOT NULL DEFAULT '0'`);
+    // iasql_db_list
+    await queryRunner.query(`
+      drop function iasql_db_list;
```

Comment:
Technically we don't need these `drop function` calls when there's a `create or replace function` call afterwards, but this is probably a bit safer in case Postgres doesn't consider them to have the same function signature.

Reply:
I mostly followed convention done from other migration files

Reply:
in the case of `iasql_db_list` we have to because the return table is changing and Postgres does not consider them the same to be replaced.